### PR TITLE
feature/primary-track-table

### PR DESCRIPTION
**Pull request by wolfbiter**
add react-table to project. implement basic PrimaryTrackTable in mix view

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
   </head>
   <body>
     <main id='main'></main>
+    <link rel="stylesheet" href="node_modules/react-table/react-table.css">
```
Comment (wolfbiter) [inline index.html:9]: is this OK?

File: mixes/components/primary-track-table.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      header: 'Key',
+      accessor: 'meta.key',
+      minWidth: 70,
+      maxWidth: 100,
+    }, {
+      header: 'Bpm',
+      accessor: 'meta.bpm',
+      minWidth: 50,
+      maxWidth: 100,
+    }]
```
Comment (wolfbiter) [inline mixes/components/primary-track-table.js:35]: so EZ

**Comment by wolfbiter** [comment]
ignore build failure - it's fixed in next PR. dont want to do the work to fix here and rebase.

File: index.html (hunk 1 above)
Comment (ahdinosaur) [reply index.html:9]: yes. or you can use `insert-css` again with [`style-resolve`](https://github.com/stackcss/style-resolve)

**Review by ahdinosaur — APPROVED** [review]
dope and legit

**Pull request merged**